test/functional/scan/test-scan-inclusive-seq.cpp

```diff
@@ -0,0 +1,19 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "test-scan-inclusive.hpp"
+
+// Sequential policy types to test
+using SequentialExecTypes = list< RAJA::seq_exec, 
+                                  RAJA::loop_exec >;
```

Comment:
Same thing here, simd_exec?

Reply:
Scans shouldn't work with simd (at least not yet).